scripts/setup/flutter.sh

```
@@ -6,26 +6,29 @@ REPO_ROOT="$( cd "$SCRIPT_DIR/../../" && pwd )"
 # ユーティリティのロード
-source "$SCRIPT_DIR/../utils/helpers.sh"
-source "$SCRIPT_DIR/../utils/logging.sh"
+source "$SCRIPT_DIR/../utils/helpers.sh" || exit 2
+
+# インストール実行フラグ
+installation_performed=false
 
 # Flutter のセットアップ
 setup_flutter() {
-    log_start "Flutter SDK のセットアップを開始します (fvm)..."
+    echo "==== Start: "Flutter SDK のセットアップを開始します (fvm)...""
```

Shellcheck (0.10.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)